sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh

```diff
@@ -139,21 +132,21 @@ then
           
       ## load conda env for samstats
       module load Anaconda3
-      source activate encodeqc
+      source $SOURCE
+      conda activate encodeqc
 
       cd ${SCRIPTDIR}
       sh ./ATACSeq/preprocessing/calcENCODEQCMetrics.sh ${sampleID}
     fi
   
     ## move log files into a folder when process(es) completed
-    cd ATACSeq/logFiles/${USER}/
-    mkdir -p ${SLURM_ARRAY_JOB_ID}
-    mv *${SLURM_ARRAY_JOB_ID}_${SLURM_ARRAY_TASK_ID}.* ${SLURM_ARRAY_JOB_ID}
-
+    
+    LOG_DIR=ATACSeq/logFiles/${USER}/${SLURM_ARRAY_JOB_ID}
+    mkdir -p $LOG_DIR
+    mv "./ATACSeq/logFiles/${USER}/ATACpreAnalysisS1${SLURM_ARRAY_JOB_ID}_${SLURM_ARRAY_TASK_ID}*" $LOG_DIR
```

Comment:
Love it, much more robust. The only problem would come from the user not setting up their log file directory (as this directory is user specific). If you were to remove the `%u` from the `#SBATCH` line (and the first reference to ${USER} on line 146) this would no longer be an issue. 
Also, from my testing, you can move the log and error files at the top of the script and it works just the same. This means that there won't be a bunch of temporary log files flooding the logFiles directory (if the script takes a long time to complete for example)

Reply:
Okay, I thought that would happen only when the process is finished, although the files are created at the beginning, so it makes sense

Reply:
Yeah, I have no idea why it works. I was using hard links before, but then I realised that just moving the files also works. I have no idea how SLURM is keeping track of where the log files are and how it is printing to them.